src/custom/state/enhancedTransactions/updater/FinalizeTxUpdater.tsx

```diff
@@ -17,6 +17,11 @@ import { GetSafeInfo, useGetSafeInfo } from 'hooks/useGetSafeInfo'
 import { useWeb3React } from '@web3-react/core'
 import { supportedChainId } from 'utils/supportedChainId'
 import { cancelOrdersBatch } from 'state/orders/actions'
+import { useSetAtom } from 'jotai'
+import { removeInFlightOrderIdAtom } from '@cow/modules/swap/state/EthFlow/ethFlowInFlightOrderIdsAtom'
+import ms from 'ms.macro'
+
+const DELAY_REMOVAL_ETH_FLOW_ORDER_ID_MILLISECONDS = ms`45s` // Delay removing the order ID since the creation time its mined (minor precaution just to avoid edge cases of delay in indexing times affect the collision detection
```

Comment:
Taking into account edge cases with network conditions, why not make this 5 or 10 min?

Reply:
Ah right, because by then the quote will be updated.
What about then 2 min to be on the safe side?